site/src/components/introduction/creating-some-data.js

```diff
@@ -0,0 +1,19 @@
+// a random number generator
+var generator = fc.data.random.walk()
+    .steps(11);
+
+// some formaters
+var valueformatter = d3.format('$f');
+var dateFormatter = d3.time.format('%b');
+
+// randomly generated sales data
```

Comment:
I was a bit confused at what generator(5) did, maybe update the comment to?

// randomly generated sales data starting at five